sdk/.env.example

```diff
@@ -1,8 +1,22 @@
+# Mnemonics used in the examples (don't ever use those for real funds)
 NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_1="endorse answer radar about source reunion marriage tag sausage weekend frost daring base attack because joke dream slender leisure group reason prepare broken river"
 NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_2="width scatter jaguar sponsor erosion enable cave since ancient first garden royal luggage exchange ritual exotic play wall clinic ride autumn divert spin exchange"
+# Seeds used in the examples (don't ever use those for real funds)
 NON_SECURE_USE_OF_DEVELOPMENT_SEED_1=0x256a818b2aac458941f7274985a410e57fb750f3a3a67969ece5bd9ae7eef5b2
 NON_SECURE_USE_OF_DEVELOPMENT_SEED_2=0x256a818b2aac458941f7274985a410e57fb750f3a3a67969ece5bd9ae7eef5b3
+# The account aliases used in the examples
+ACCOUNT_ALIAS_1="Alice"
+ACCOUNT_ALIAS_2="Bob"
+# The wallet database folder used in the examples
+WALLET_DB_PATH="./example.walletdb"
+# The Stronghold snapshot file used in the examples
+STRONGHOLD_SNAPSHOT_PATH="./example.stronghold"
+# The password to the Stronghold snapshot file
 STRONGHOLD_PASSWORD="some_hopefully_secure_password"
+# The node to issue transactions with in the examples
```

Comment:
with in the examples sounds weird

Reply:
it has the same structure as "the guys I went to school with in my teenage years"   which doesnt sound weird to me tbh

Reply:
Ok maybe it's just me
Another thing, all env values are for the examples, so why mention examples above most of them? If you want to mention that they are for the examples, I would just put a single comment on top for all

Reply:
fair enough :+1: